main.nf

```diff
@@ -0,0 +1,932 @@
+        TMP_DIR=tmp \\
+        VALIDATION_STRINGENCY=SILENT \\
+        REMOVE_DUPLICATES=false \\
+        ASSUME_SORTED=false \\
+        MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 \\
+        MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 \\
+        SORTING_COLLECTION_SIZE_RATIO=0.25 \\
+        READ_NAME_REGEX=\"[a-zA-Z0-9]+:[0-9]:([0-9]+):([0-9]+):([0-9]+).*\" \\
+        OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 \\
+        VERBOSITY=INFO \\
+        QUIET=false \\
+        COMPRESSION_LEVEL=5 \\
+        CREATE_INDEX=false \\
+        MAX_RECORDS_IN_RAM=500000 \\
+        CREATE_MD5_FILE=false \\
+        GA4GH_CLIENT_SECRETS=''
+
+    # Print version number to standard out
+    echo "Picard version "\$(picard  MarkDuplicates --version 2>&1)
+    """
+}
+
+
+/*
+ * Step 5 - Recalibrate BAM file with known variants and BaseRecalibrator
+ * 
+*/ 
+process recal_bam_files {
+    tag "${name}"
+
+    input:
+    set val(name), file(markdup_bam) from samples_markdup_bam
+
+    output:
+    set val(name), file("${name}_recal.bam"), file("${name}_recal.bai") into samples_recal_bam
+    file '.command.log' into gatk_stdout
+
+    script:
+    """
+    gatk -T BaseRecalibrator \\
+        -I $markdup_bam \\
+        -R $params.gfasta \\
+        -o ${name}_table.recal \\
+        -cov ReadGroupCovariate \\
+        -cov QualityScoreCovariate \\
+        -cov CycleCovariate \\
+        -cov ContextCovariate \\
+        -U \\
+        -OQ \\
+        --default_platform illumina \\
+        --knownSites $params.dbsnp \\
+        -l INFO
+
+    gatk -T PrintReads \\
+        -BQSR ${name}_table.recal \\
+        -I $markdup_bam \\
+        -R $params.gfasta \\
+        -o ${name}_recal.bam \\
+        -baq RECALCULATE \\
+        -U \\
+        -OQ \\
+        -l INFO
+
+    # Print version number to standard out
+    echo "GATK version "\$(gatk --version 2>&1)
+    """
+}
+
+
+/*
+ * Step 6 - Realign BAM file around indels
+ * 
+*/ process realign {
+    tag "${name}"
+    publishDir "${params.outdir}/realigned/alignment", mode: 'copy',
+        saveAs: {filename -> filename.replaceFirst(/realign/, "sorted_dupmarked_recalibrated_realigned")}
+
+    input:
+    set val(name), file(recal_bam), file(recal_bam_ind) from samples_recal_bam
+
+    output:
+    set val(name), file("${name}_realign.bam"), file("${name}_realign.bai") into (bam_vcall, bam_metrics)
+
+    script:
+    """
+    gatk -T RealignerTargetCreator \\
+        -I $recal_bam \\
+        -R $params.gfasta \\
+        -o ${name}_realign.intervals \\
+        --known $params.dbsnp \\
+        -l INFO
+
+    gatk -T IndelRealigner \\
+        -I $recal_bam \\
+        -R $params.gfasta \\
+        -targetIntervals ${name}_realign.intervals \\
+        -o ${name}_realign.bam \\
+        -l INFO
+    """
+}
+
+/*
+ * Step 7 - Determine quality metrics of mapped BAM files using QualiMap 2
+ * 
+*/ 
+process qualiMap {
+    tag "${name}"
+    publishDir "${params.outdir}/qualimap/qualimap", mode: 'copy'
```

Comment:
`/qualimap/qualimap` -> `/qualimap`

Reply:
Removed in 646bcb56e1953ac92546ef0900cb343e86ee2cab